### PR TITLE
[Don't merge] Add the trade desk connector API docs

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

Repo PR: https://github.com/wso2-extensions/esb-connector-thetradedesk/pull/1

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -700,6 +700,9 @@ nav:
           - Facebook Ads Connector:
               - Facebook Ads Overview: reference/connectors/facebook-ads-connector/facebook-ads-overview.md
               - Facebook Ads Reference: reference/connectors/facebook-ads-connector/facebook-ads-configuration.md
+          - The Trade Desk Connector:
```
Comment (DinithiDiaz) [inline en/mkdocs.yml:703]:
```suggestion
          - Trade Desk Connector:
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration.md [hunk 2]
```diff
@@ -0,0 +1,610 @@
+# The Trade Desk Connector Reference
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration.md:1]:
```suggestion
# Trade Desk Connector Reference
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# The Trade Desk Connector Overview
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:1]:
```suggestion
# Trade Desk Connector Overview
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+The Trade Desk Connector allows you to access The Trade Desk API. This lets you create, update, and delete programmatic ad campaigns and ads.
+
+To see The Trade Desk Connector, navigate to the [connector store](https://store.wso2.com) and search for "trade desk".
+
+<img src="{{base_path}}/assets/img/integrate/connectors/the-trade-desk.png" title="The Trade Desk Connector Store" width="200" alt="The Trade Desk Connector Store"/>
+
+## Compatibility
+
+| Connector Version | Supported product versions |
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:13]:
```suggestion
| Connector version | Supported product versions |
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+<img src="{{base_path}}/assets/img/integrate/connectors/the-trade-desk.png" title="The Trade Desk Connector Store" width="200" alt="The Trade Desk Connector Store"/>
+
+## Compatibility
+
+| Connector version | Supported product versions |
+| ------------- |-------------|
+| 1.0.0    | MI 4.3.0 |
+
+## The Trade Desk Connector documentation
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:17]:
```suggestion
## Trade Desk Connector documentation
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+## Compatibility
+
+| Connector version | Supported product versions |
+| ------------- |-------------|
+| 1.0.0    | MI 4.3.0 |
+
+## Trade Desk Connector documentation
+
+* **[The Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for the Trade Desk Connector.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:19]:
```suggestion
* **[Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for the Trade Desk Connector.
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Trade Desk Connector Overview
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:1]:
```suggestion
# The Trade Desk Connector Overview
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Compatibility
+
+| Connector version | Supported product versions |
+| ------------- |-------------|
+| 1.0.0    | MI 4.3.0 |
+
+## The Trade Desk Connector documentation
+
+* **[Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for the Trade Desk Connector.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:19]:
```suggestion
* **[The Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for the Trade Desk Connector.
```

File: en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+## Compatibility
+
+| Connector version | Supported product versions |
+| ------------- |-------------|
+| 1.0.0    | MI 4.3.0 |
+
+## The Trade Desk Connector documentation
+
+* **[The Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for the Trade Desk Connector.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/the-trade-desk-connector/the-trade-desk-overview.md:19]:
```suggestion
* **[The Trade Desk Connector Reference]({{base_path}}/reference/connectors/the-trade-desk-connector/the-trade-desk-configuration/)**: This documentation provides a reference guide for The Trade Desk Connector.
```

**Comment by DinithiDiaz** [comment]
Hi @aashikam, could you kindly send these changes to the [`4.3.0`](https://github.com/wso2/docs-mi/tree/4.3.0) branch, which is the deployed branch for the latest docs?

**Comment by aashikam** [comment]
> Hi @aashikam, could you kindly send these changes to the [`4.3.0`](https://github.com/wso2/docs-mi/tree/4.3.0) branch, which is the deployed branch for the latest docs?

Hi, Thanks for the review. We wont be releasing this connector, therefore these docs cannot go live. I will close this PR.

**Pull request closed without merging**